samples/com.google.android.play/play-services-assetpack/AssetPackSample/AssetPackSample.csproj

```diff
@@ -129,13 +55,5 @@
       <ReferenceOutputAssembly>false</ReferenceOutputAssembly>
     </ProjectReference>
   </ItemGroup>
-  <Import Project="$(MSBuildExtensionsPath)\Xamarin\Android\Xamarin.Android.CSharp.targets" />
-  <Import Project="..\Targets\DynamicFeature.targets" />
```

Comment:
This is needed for the dynamic features to work.

Reply:
Also missing the `AssetPacks` and Targets folders in the directory above.

Reply:
So all of this https://github.com/xamarin/GooglePlayServicesComponents/tree/main/samples/com.google.android.play/play-services-assetpack